cmd/ndm_daemonset/filter/osdiskexcludefilter.go

```diff
@@ -17,9 +17,10 @@ limitations under the License.
 package filter
 
 import (
-	"k8s.io/klog"
 	"strings"
 
+	"k8s.io/klog"
```

Comment:
Can you keep the import statement such that vendored packages are the last section like [this](https://github.com/openebs/node-disk-manager/blob/503ec7fb835ee439687d39ab91599a63301a2233/cmd/ndm_daemonset/grpc/api.go#L19)